schema/fs.go

```diff
@@ -379,735 +452,9107 @@ hR+PkX5zgcX30ucJvpgCXqksmwP73wn8U2qZo81xwnEy+6yyZ6Vexsl79az+DgAA//850zAaggUAAA==
 
 	"/fs.go": {
```

Comment:
Ahh, recursive `http.FileSystem`!  And not new to this PR either :p.  In fact, it looks like it goes all the way back to #65.